knox/models.py

```diff
@@ -19,7 +19,7 @@ def create(self, user, expiry=knox_settings.TOKEN_TTL):
 
         instance = super(AuthTokenManager, self).create(
             token_key=token[:CONSTANTS.TOKEN_KEY_LENGTH], digest=digest,
-            salt=salt, user=user, expiry=expiry)
+            salt=salt, user=user, expiry=expiry, **kwargs)
```

Comment:
Can you explain this change?

Reply:
Ok but then you need to allow those params to be passed everywhere create is used in the code and this is not the case right now.